test/utility_clopper_pearson_interval_test.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 Hans Dembinski
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <boost/core/lightweight_test.hpp>
+#include <boost/histogram/utility/clopper_pearson_interval.hpp>
+#include <limits>
+#include "is_close.hpp"
+#include "throw_exception.hpp"
+
+using namespace boost::histogram::utility;
+
+int main() {
+
+  // const double deps = std::numeric_limits<double>::epsilon();
+  const double deps = 0.1;
```

Comment:
This requires a comment why deps is so large.

Reply:
I see your improved implementation now covers 0.001. I will take a look on how you did it.

Reply:
You used alpha = CL =0.68. instead of alpha = 1 - CL =0.32.